VideoLocker/src/main/java/org/edx/mobile/module/registration/model/RegistrationOption.java

```diff
@@ -19,6 +19,19 @@ public boolean isDefaultValue() {
         return defaultValue;
     }
 
+    public boolean equals(Object object){
+        if ( object == null || !(object instanceof RegistrationOption) )
+            return false;
+        RegistrationOption otherObject = (RegistrationOption)object;
+        return otherObject.name != null && otherObject.name.equals( name )
+                && otherObject.value != null && otherObject.value.equals( value );
+    }
+
+    public int hashCode(){
+        int hashcode = name == null ? 0 : name.hashCode() / 2;
```

Comment:
We're on java 7 right? I think you can use ``Objects.hash`` here. If we're not, then it looks like Arrays.hashCode should work.